modular_skyrat/modules/reagent_forging/code/forge.dm

```diff
@@ -178,14 +184,14 @@
 		to_chat(user, span_notice("[src] requires you to be more experienced!"))
 		return
 	if(user_smithing_skill >= SKILL_LEVEL_APPRENTICE)
-		goliath_ore_improvement = 3
+		goliath_ore_improvement = MAX_UPGRADE_GOLIATH
 		forge_level = FORGE_LEVEL_ONE
 	if(user_smithing_skill >= SKILL_LEVEL_EXPERT)
-		sinew_lower_chance = 100
-		current_sinew = 10
+		sinew_lower_chance = MAX_UPGRADE_SINEW * 10 //100, just written funny!
```

Comment:
That feels a bit weird, I'd make it its own define or something, because this isn't really related to the amount of upgrades from sinew you can do, and is prone to breaking if that `MAX_UPGRADE_SINEW` define is changed.

Reply:
Actually, looking back on this, just make the define be 100, that'd be clearer.

Reply:
people would see the 100 and think they would need 100 sinew.